charts/private-action-runner/README.md.gotmpl

```diff
@@ -22,69 +22,6 @@ helm repo update
 1. Go to the [Private Action Runner tab](https://app.datadoghq.com/workflow/private-action-runners).
 2. Create a new Private Action Runner and follow the instructions for Kubernetes.
 
-## Use this chart with connection credentials
```

Comment:
i think it's worth documenting this. credentials are a lot easier to set up now in the chart but i think at least having some copy about how to link the chart credentials to a connection would be useful.

Reply:
I agree I think especially clarifying what they should set for `Path to Secret Location` in the connection UI would be helpful